rasa/shared/core/domain.py

```diff
@@ -922,6 +873,7 @@ def cleaned_domain(self) -> Dict[Text, Any]:
                 del slot["initial_value"]
             if slot["auto_fill"]:
                 del slot["auto_fill"]
+            # TODO!!!!
```

Comment:
⏫  still relevant?

Reply:
removed it, but made a todo in our channel